### PR TITLE
redirect to https://icesat-2.hackweek.io

### LINKED ISSUE #11: remove headshot image files from assets/images and use GitHub links instead

### LINKED ISSUE #15: redirect to new website
This repository carries over from previous events and creates https://icesat-2hackweek.github.io, which is now a stale page in favor of https://icesat-2.hackweek.io

Problem is the old page is the top google hit. And we're tired of ending up there ;) 

Instead of taking this page down completely I think we can just do a redirect as described here:
https://dev.to/steveblue/setup-a-redirect-on-github-pages-1ok7  

Ultimately we want a year identifier in the URL. see also https://github.com/uwhackweek/jupyterbook-template/issues/85)

### PR DESCRIPTION
**Pull request by scottyhq**
closes #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # ICESat-2 Hackweek web site
 
+*NOTE:* this website (https://icesat-2hackweek.github.io) now just redirects to icesat-2hackweek.github.io. The instructions below apply to previous version os the website, corresponding to the the 2022.03.01 tag: https://github.com/ICESAT-2HackWeek/icesat-2hackweek.github.io/tree/2022.03.01
```
Comment (scottyhq) [inline README.md:3]: if for some reason you want to resurrect the website

Comment (jomey) [reply README.md:3]: If I follow the example correctly, shouldn't this say:
```
... now just redirects to https://icesat-2.hackweek.io/
```

This is how the `index.html` meta tag is setup

Comment (JessicaS11) [reply README.md:3]:
```suggestion
*NOTE:* this legacy website (https://icesat-2hackweek.github.io) automatically redirects to the newer [icesat-2hackweek.github.io](https://icesat-2.hackweek.io/). The instructions below apply to the previous version of the website, corresponding to the the 2022.03.01 tag: https://github.com/ICESAT-2HackWeek/icesat-2hackweek.github.io/tree/2022.03.01. icesat-2.hackweek.io is generated by a single, templated repository that includes all event materials.
```

**Comment by JessicaS11** [comment]
Looking good to me. Should we go even more bare bones and remove some of the other files from the repo (e.g. github actions, the empty contributors file, etc.)?

**Comment by scottyhq** [comment]
> Should we go even more bare bones and remove some of the other files from the repo (e.g. github actions, the empty contributors file, etc.)?

Just did in last commit, but left the github actions in there because they still work

**Pull request merged**